tests/unit/backend/utils/field-validation/attachment-validation.spec.js

```diff
@@ -109,14 +109,51 @@ describe('Attachment validation', () => {
     })
   })
 
-  it('should disallow responses submitted for hidden fields', () => {
-    const formField = makeField(fieldId, '3')
-    const response = makeResponse(fieldId, Buffer.alloc(2000000))
-    response.isVisible = false
-    const validateResult = validateField(formId, formField, response)
-    expect(validateResult.isErr()).toBe(true)
-    expect(validateResult._unsafeUnwrapErr()).toEqual(
-      new ValidateFieldError('Attempted to submit response on a hidden field'),
-    )
+  describe('check for responses on hidden fields should disallow responses submitted for hidden fields', () => {
+    it('when response contains file content', () => {
```

Comment:
```suggestion
  describe('check for responses on hidden fields', () => {
    it('should disallow responses submitted for hidden fields when response contains file content', () => {
```

convention usually dictates that `it` should be followed by `should ...`